### PR TITLE
Removes Secglasses (in favor of aviators)

### PR DESCRIPTION
**Pull request by ParadoxSpace**
At request of, surprisingly, AmoryBlaine. Also fixes all other usages of the hudglasses, making sure aviators work for them. Also, fixes the secpatch, medpatch, and medviators not working right.

### CODE REVIEW

File: code/controllers/subsystems/traumas.dm [hunk 1]
```diff
@@ -53,7 +53,7 @@ var/datum/controller/subsystem/traumas/SStraumas
 
 					   "security"  = typecacheof(list(/obj/item/clothing/under/rank/security, /obj/item/clothing/under/rank/warden, /obj/item/clothing/head/beret/sec,
 											 	 /obj/item/clothing/under/rank/head_of_security, /obj/item/clothing/under/det, /obj/item/clothing/glasses/hud/security,
-												 /obj/item/weapon/melee/baton, /obj/item/weapon/gun/energy/taser, /obj/item/weapon/handcuffs, /obj/item/clothing/glasses/sunglasses/sechud,
+												 /obj/item/weapon/melee/baton, /obj/item/weapon/gun/energy/taser, /obj/item/weapon/handcuffs, /obj/item/clothing/glasses/sunglasses/sechud/aviator,
```
Comment (Sindorman) [inline code/controllers/subsystems/traumas.dm]: Keep the original, but add aviators to the list

Comment (ParadoxSpace) [reply code/controllers/subsystems/traumas.dm]: ok fine done

**Comment by ParadoxSpace** [comment]
!review

**Comment by Kaedwuff** [comment]
Yuck, no. Aviators look stupid we already have them as an option if you really like them.

**Comment by ParadoxSpace** [comment]
How are they stupid?

**Comment by Kaedwuff** [comment]
> How are they stupid?

I don't like the way they look, and we already have a choice between the one we like. There is zero reason to remove the regular HUD glasses except pigeonhole the option you want to be in the game.

The last time the aviators were introduced into the game someone tried to remove the other option at the same time and we ended up compromising with both of them being available. Why are you revisiting this decision without consulting the community?

**Comment by TehFlaminTaco** [comment]
I thought we were keeping hudglasses as a rarer option as a caveat for adding the nerf that is HUD Aviators. -1

**Comment by Sindorman** [comment]
Don't post on PR anymore about feedback. Let's have feedback thread. Paradox please make feedback thread and post link here

**Comment by skull132** [comment]
@Sindorman Feedback thread for this will serve no purpose. The only responses you'll get are, "I like X," or "I like Y."

**Comment by NiftyBaby** [comment]
I severely dislike this change. Aviators look horrid. Can we not ruin things that others like just to push our own agendas?

**Comment by Sindorman** [comment]
@NiftyBaby stop non-code comments.

**Comment by NiftyBaby** [comment]
Find a way for us to leave feedback if we can't make a feedback thread.

**Comment by Sindorman** [comment]
@NiftyBaby Read above, the feedback thread will not be required. chrome_xDLGlw5fmM

**Comment by skull132** [comment]
https://forums.aurorastation.org/topic/12128-removing-secglasses-feedback-thread/?tab=comments#comment-115765

Per Arrow's note, removing one in favour of the other is not worth it at present.

If you want to standardize the mechanics, then please do so in a PR that retains both sprites. You're also free to put a PR up touching up the old icons to look higher quality. But at present, this one is hitting the bin.

**Pull request closed without merging**